src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NugetApiPackageManager.cs

```diff
@@ -250,7 +250,7 @@ private async Task<IEnumerable<IPackageSearchMetadata>> GetPackageMetadataAsync(
                 PackageMetadataResource resource = await repository.GetResourceAsync<PackageMetadataResource>(cancellationToken).ConfigureAwait(false);
                 IEnumerable<IPackageSearchMetadata> foundPackages = await resource.GetMetadataAsync(
                     packageIdentifier,
-                    includePrerelease: false,
+                    includePrerelease: includePrerelease,
```

Comment:
I'm commenting this whole commit:
I suggested this behavior, but now I wonder what will happen when user installs preview, and in next step calls `dotnet new --check-updates`? And preview is newer then latest stable?